modular_skyrat/modules/encounters/code/enclave_pirates.dm

```diff
@@ -124,6 +115,13 @@
 	suffix = "enclave"
 	name = "pirate ship (NRI Enforcer-Class Starship)"
 
+/area/shuttle/pirate/nri
+	name = "NRI Starship"
```

Comment:
Is this intentionally still using the "pirate" shuttle ID?

Reply:
Yes, or otherwise their objective doesn't show up and register at the end of the round.